README.md

```diff
@@ -192,10 +196,10 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
  - [Mongoid](https://github.com/mongodb/mongoid) - ODM framework
 
 ### Rust
- - [mongodb-rust-driver](https://github.com/mongodb/mongo-rust-driver) - Official Rust driver
+ - [mongodb-rust-driver](https://github.com/mongodb/mongo-rust-driver) - Official MongoDB Rust driver
 
 ### Scala
- - [mongo-scala-driver](https://github.com/mongodb/mongo-scala-driver) - Official Scala driver
+ - [mongo-java-driver](https://github.com/mongodb/mongo-java-driver) - Official MongoDB Scala driver
```

Comment:
We probably want to remove that one as well

Reply:
I see how the name can be confusing, but the repo contains up-to-date, MongoDB, Inc. driver code for the Java Sync, Java Reactive Streams, and Scala drivers. Or did I misinterpret the reason why this should be removed?

Reply:
Ah, if it contains scala specific code then it can definitely stay, but we should say it in the description somehow. My understanding was that the Scala driver just got deprecated in favor of the Java driver as-is.